.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  aws-cli: circleci/aws-cli@4.1.3
```

Comment:
Downgrade?

Reply:
Yeah, just getting everything to be the same consistent version across repos for now and using a version I know works via testing.